### DISCUSSION TITLE
Bug Report: "All" option in language filter in server browser doesn't work

### DISCUSSION THREAD

**Discussion by Abkim27** [FakeFishGames/Barotrauma#14859] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Pressing the "All" option in language filter just doesn't do anything. It's being checked while you holding the LMB, but right after you release the button it just returns to unchecked form and doesn't do anything

### Reproduction steps

1.Enter the game
2.Enter the server browser
3.Uncheck something from language filter, so "All" option is unchecked (though, if you press it while it's checked, it still doesn't do anything)
4.Press the "All" option

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.17.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Language selection now works when selecting any or all languages.
Closing discussion as resolved.
